tests/storage/largeblock/test_largeblock_sr.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+from lib.common import wait_for, vm_image
+from tests.storage import try_to_create_sr_with_missing_device, vdi_is_open
+
+# Requirements:
+# - one XCP-ng host >= 8.3 with an additional unused 4KiB disk for the SR
```

Comment:
To enforce the version requirement, you can include the `host_at_least_8_3` fixture.

See zfsvol tests for an example.